### DISCUSSION TITLE
Wrong unit for heatpump entities "hppl1" and "hpph1"

### DISCUSSION THREAD

**Discussion by KLiebrecht1** [emsesp/EMS-ESP32#2020] [General]
The values for "hppl1" and "hpph1" have the wrong unit. Not temperature but pressure.
Can you tell me which telegram-id these values are in?
Then I can search for the data format.

ems-esp Version 3.7.0-dev.38
Device HM200
Device-ID 0x53

{
  "name": "hppl1",
  "fullname": "Niederdruckfühler (PL1)",
  "circuit": "",
  "value": 13.3,
  "type": "number",
  "uom": "°C",
  "readable": true,
  "writeable": false,
  "visible": true
}

{
  "name": "hpph1",
  "fullname": "Hochdruckfühler (PH1)",
  "circuit": "",
  "value": 13,
  "type": "number",
  "uom": "°C",
  "readable": true,
  "writeable": false,
  "visible": true
}

**Comment by proddy** [comment]
Both from telegram 0x48F

hppl1 is data offset 26, hpph1 at 28

**Comment by KLiebrecht1** [comment]
Thanks for your reply, but there is no telegram 0x48F.
'Watch on 48f' for about half an hour gives no result.
[emsesp_system_undefined.txt](https://github.com/user-attachments/files/17024067/emsesp_system_undefined.txt)

**Reply by proddy** [reply]
My mistake, was looking at the boiler. For a heat pump try

0x09A0 and `hppl1` at 18 (low pressure), `hpph1` at 20 (high pressure)

**Reply by MichaelDvP** [reply]
as reported in #500 these are temperatures on high and low pressure side.
I think the german translation needs be better.

**Reply by KLiebrecht1** [reply]
Ah, now it makes sense. I thought 'hppl1' and 'hpph1' included the pressure values, because there are no sensors for low and high pressure temperature in the Buderus circuit diagram.
Only the pressure sensors 'JR0' and 'JR1'.
The pressure values for 'JR0' and 'JR1' must therefore still be present in some telegram. 

The reason for that is that i got this information from Buderus:
"Unterschieden werden 4 Abtauarten, die von der Betriebsart, der Zulufttemperatur TL2 (ähnlich der Aussentemperatur) und des Druckes am Niederdrucksensor JR0 abhängig sind."

Thanks for your efforts

**Reply by MichaelDvP** [reply]
> The pressure values for 'JR0' and 'JR1' must therefore still be present in some telegram.

maybe there is also pressure, but JR0 and JR1 are the temperature-values that was reported in #500: See:
https://user-images.githubusercontent.com/77983933/219854978-ce798e35-8b55-4f7d-aa1f-6dc4a99571c4.png
